odxtools/dtcdop.py

```diff
@@ -30,7 +76,7 @@ class DtcDop(DopBase):
     physical_type: PhysicalType
     compu_method: CompuMethod
     dtcs_raw: List[Union[DiagnosticTroubleCode, OdxLinkRef]]
-    linked_dtc_dop_refs: List[OdxLinkRef]
+    linked_dtc_dops_raw: List[LinkedDtcDop]
```

Comment:
The linked dtcs don't affect serialization logic?

Reply:
they do, that's why `printDOP.xml.jinja2` is modified by this commit...

Reply:
I am asking if they affect encoding / decoding messages

Reply:
they do not directly affect en-/decoding: just make additional DTCs appear for value parameters that use a DTC-DOP, so additional DTCs can be decoded.